src/briefcase/integrations/android_sdk.py

```diff
@@ -897,8 +900,8 @@ def start_emulator(self, avd):
                 self.sleep(2)
 
         # Phase 2: Wait for the boot process to complete
-        with self.command.input.wait_bar("Booting..."):
-            while not adb.has_booted():
+        while not adb.has_booted():
+            with self.command.input.wait_bar("Booting emulator..."):
```

Comment:
Flip flop to avoid saying we're waiting on the device to boot when it's already booted.

Reply:
I clearly did not properly test this. Thanks.